### ISSUE TITLE
v0.9.2.9 - DS918+ 7.0.1-42218:  Error "unified diff" when attempting to patch

### ISSUE THREAD

**Issue by xiongxai** [pocopico/tinycore-redpill#193]
Trying to install DSM 7.0.1 using TCRP v0.9.2.9 on ESXi 6.7.  Set up TCRP vmdk on SATA controller 0 and 1 new/empty vmdk on SATA controller 1.  I'm attempting to build with ds918p-7.0.1-42218 and get the following error when I try to perform the build:

Initial configuration:
./rploader.sh fullupgrade
./rploader.sh serialgen DS918+ realmac
./rploader.sh satamap

Build command and the error output:
./rploader.sh build ds918p-7.0.1-42218

[#] Apply patches to /home/tc/redpill-load/build/1668198110/rd-ds918p_42218-unpacked...
[!] One of the patches - /home/tc/redpill-load/config/_common/ramdisk-005-disable-disabled-ports.patch - failed to apply

Hmm...  Looks like a unified diff to me...
The text leading up to this was:
--------------------------
|--- a/usr/syno/web/webman/get_state.cgi
|+++ b/usr/syno/web/webman/get_state.cgi
--------------------------
patching file usr/syno/web/webman/get_state.cgi
Using Plan A...
Hunk #1 FAILED at 1.
1 out of 1 hunk FAILED -- saving rejects to file usr/syno/web/webman/get_state.cgi.rej
done

*** Process will exit ***
FAILED : Loader creation failed check the output for any errors

**Comment by pocopico** [comment]
You can edit the config file and remove that specific patch line

**Comment by xiongxai** [comment]
Is this the correct config file?  /home/tc/config/DS918+/7.0.1-42218/config.json

I removed "@@@COMMON@@@/ramdisk-005-disable-disabled-ports.patch" and still getting the same error on the build.

**Comment by pocopico** [comment]
Yes that’s the correct file. Can you try another version ? 42661 or 42961 ?

**Comment by xiongxai** [comment]
@pocopico - I specifically want to install 42218 so that I can utilize AME.  42962 works for me but the AME in 7.1 requires validation with synology.com

I was able to get the build to complete by modifying the following file after a failed build:  /home/tc/redpill-load/config/DS918+/7.0.1-42218/config.json

Not sure why this file which appears to be used during the build did not get copied from /home/tc/config ?

Anyway my issue is resolved.  Thanks!

**Comment by tekima** [comment]
> @pocopico - I specifically want to install 42218 so that I can utilize AME. 42962 works for me but the AME in 7.1 requires validation with synology.com
> 
> I was able to get the build to complete by modifying the following file after a failed build: /home/tc/redpill-load/config/DS918+/7.0.1-42218/config.json
> 
> Not sure why this file which appears to be used during the build did not get copied from /home/tc/config ?
> 
> Anyway my issue is resolved. Thanks!

I have got exactly the same problem, can you share how you can overcome it. I also want to use 7.0.1 only.
